### PR TITLE
feat: add qr code button

### PR DESCRIPTION
**Pull request by qiweiii**
related: https://github.com/Magickbase/godwoken_explorer/issues/813

preview: https://godwoken-explorer-ui-git-feat-qr-code-nervosnet.vercel.app/account/0x9f2a42a167212b7035cc8e7cdafe0a09232d6c18

image

### CODE REVIEW

File: components/QRCodeBtn/index.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+const QRCodeBtn: React.FC<{ content: string }> = ({ content }) => {
+  const [t] = useTranslation('common')
+  const [open, setOpen] = useState(false)
+  const [data, setData] = useState('')
+  const theme = useTheme()
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'))
+
+  const handleOpen = async () => {
+    const dataUrl = await generateQR(content)
```
Comment (Keith-CY) [inline components/QRCodeBtn/index.tsx:21]: `quality` attribute could be set higher

**Review by Keith-CY — APPROVED** [review]

**Comment by FrederLu** [comment]
image

@qiweiii The save prompt is not displayed after saving on the Android phone.

**Comment by qiweiii** [comment]
> image
> 
> @qiweiii The save prompt is not displayed after saving on the Android phone.

Not implemented, as I found it is impossible to detect when file is successfully saved to device in a web app

**Comment by FrederLu** [comment]
@Kirl70 
The prompt information cannot be implemented on the mobile phone. Currently, it can be detected that the user starts to download, but it cannot be detected whether the download is completed.

> > image
> > @qiweiii The save prompt is not displayed after saving on the Android phone.
> 
> Not implemented, as I found it is impossible to detect when file is successfully saved to device in a web app

**Comment by Kirl70** [comment]
> @Kirl70 The prompt information cannot be implemented on the mobile phone. Currently, it can be detected that the user starts to download, but it cannot be detected whether the download is completed.
> 
> > > image
> > > @qiweiii The save prompt is not displayed after saving on the Android phone.
> > 
> > 
> > Not implemented, as I found it is impossible to detect when file is successfully saved to device in a web app

OK, if the technology can't be implemented, then there is no need to prompt

**Review by FrederLu — APPROVED** [review]

**Pull request merged**